index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="body-wrapper">
+        <nav class="header-flex">
+            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
+            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
+        </nav>
+        <div class="container-flex">
+            <div class="intro-flex">
+                <h1>Omar Mohammad</h1>  
+                <h2>Software Developer</h2>        
+            </div>
```

Comment:
- [ ] Great job using `<h1>` tag here 💯, but remember according to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, change the font color and add horizontal margin between these elements to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177020983-44c58471-9868-4b96-9248-52a726ae24b1.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/177020967-3f00e0b7-c8fb-4291-b76a-3682e43fff92.PNG" width="500" /> |

Reply:
OK.